Beeper/barcelona-mautrix/SendMessageCLICommand.swift

```diff
@@ -29,6 +30,9 @@ class SendMessageCLICommand: Command {
     @Param var chatGuid: String
     @Param var message: String?
 
+    @Flag("--force-available")
+    var overwriteAvailability: Bool
```

Comment:
@itsjunetime Switched this to a flag, seems to work better and is probably a better fit than another param.